app/src/androidTest/java/com/google/samples/apps/topeka/activity/SignInActivityTest.java

```diff
@@ -65,20 +72,142 @@ public void checkFab_initiallyNotDisplayed() {
     }
 
     @Test
+    public void signIn_checkIfFirstNameEditTextIsInitiallyEmpty() {
```

Comment:
well, I thought that signIn prefix is like namespace of tests showing of which part test is about.
In my Espresso tests I'm usingformats like <code>check_01_ifToolbarIsProperlyDisplayed</code> or <code>perform_doubleClickOnLoginButton</code>